templates/terraform/resource.erb

```diff
@@ -217,21 +217,24 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
 
 <%  if !object.async.nil? && object.async.allow?('create') -%>
 <%    if object.async.is_a? Api::PollAsync -%>
-    waitURL, err := replaceVars(d, config, "<%= "{{#{object.__product.name}BasePath}}#{object.self_link_uri}" -%>")
+    pollUrl, err := replaceVars(d, config, "<%= "{{#{object.__product.name}BasePath}}#{object.self_link_uri}" -%>")
     if err != nil {
         return err
     }
 
-    err = <%= client_name_camel -%>PollingWaitTime(
-    config, res,<% if has_project -%> project, <% end -%> waitURL, "Creating <%= object.name -%>",
-        int(d.Timeout(schema.TimeoutCreate).Minutes()))
-
+    err = PollingWaitTime(config, <%= client_name_camel -%>PollWaiter(res), <% if has_project -%> project, <% end -%> pollUrl, "Creating <%= object.name -%>",
+        time.Minute * <%= object.async.operation.timeouts.insert_minutes %>)
```

Comment:
Using this means that it'll always be the default timeout instead of a user-configured override. Is that intentional?

Reply:
yes - as spew mentioned, we want this value to be very small, but the original timeout was 6 minutes. If you feel differently, I can change it back, but I don't think this timeout should be the same timeout. This was also more of an issue when I was trying the match-state poll and finding out the state differs for subscriptions, so we can't directly match it.

Reply:
Taking a look at the generated cloud run code domain mapping though, the only place where the user-specified create timeout gets used is in the initial create call, even though there's still time spent polling after for a successful create. That's what seems weird to me.

Reply:
I'll just change it back and keep the timeout the same. It's probably not an issue with the 404 checker, and theoretically people have been willing to wait 6 minutes already for this resource.